src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/impl/AbstractAuditLog.java

```diff
@@ -118,77 +100,12 @@
-                DEFAULT_DISABLED_CATEGORIES));
-
-        disabledTransportCategories = AuditCategory.parse(getConfigList(
-                settings,
-                ConfigConstants.OPENDISTRO_SECURITY_AUDIT_CONFIG_DISABLED_TRANSPORT_CATEGORIES,
-                DEFAULT_DISABLED_CATEGORIES));
-
-        logRequestBody = settings.getAsBoolean(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_LOG_REQUEST_BODY, true);
-        resolveIndices = settings.getAsBoolean(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_RESOLVE_INDICES, true);
-
-        ignoredAuditUsers = new ArrayList<>(settings.getAsList(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_IGNORE_USERS, defaultIgnoredUsers));
-
-        if(ignoredAuditUsers.size() == 1 && "NONE".equals(ignoredAuditUsers.get(0))) {
-            ignoredAuditUsers.clear();
-        }
-
-        if (ignoredAuditUsers.size() > 0) {
-            log.info("Configured Users to ignore: {}", ignoredAuditUsers);
-        }
-
-        ignoredComplianceUsersForRead = new ArrayList<>(settings.getAsList(ConfigConstants.OPENDISTRO_SECURITY_COMPLIANCE_HISTORY_READ_IGNORE_USERS, defaultIgnoredUsers));
-
-        if(ignoredComplianceUsersForRead.size() == 1 && "NONE".equals(ignoredComplianceUsersForRead.get(0))) {
-            ignoredComplianceUsersForRead.clear();
-        }
-
-        if (ignoredComplianceUsersForRead.size() > 0) {
-            log.info("Configured Users to ignore for read compliance events: {}", ignoredComplianceUsersForRead);
-        }
-
-        ignoredComplianceUsersForWrite = new ArrayList<>(settings.getAsList(ConfigConstants.OPENDISTRO_SECURITY_COMPLIANCE_HISTORY_WRITE_IGNORE_USERS, defaultIgnoredUsers));
-
-        if(ignoredComplianceUsersForWrite.size() == 1 && "NONE".equals(ignoredComplianceUsersForWrite.get(0))) {
-            ignoredComplianceUsersForWrite.clear();
-        }
-
-        if (ignoredComplianceUsersForWrite.size() > 0) {
-            log.info("Configured Users to ignore for write compliance events: {}", ignoredComplianceUsersForWrite);
-        }
-
-        ignoreAuditRequests = settings.getAsList(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_IGNORE_REQUESTS, Collections.emptyList());
-        if (ignoreAuditRequests.size() > 0) {
-            log.info("Configured Requests to ignore: {}", ignoreAuditRequests);
-        }
-
-        this.excludeSensitiveHeaders = settings.getAsBoolean(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_EXCLUDE_SENSITIVE_HEADERS, true);
-    }
-
-    private static List<String> getConfigList(final Settings settings,
-                                              final String key,
-                                              final List<String> defaultList) {
-        List<String> list = settings.getAsList(key, defaultList);
-        if (list.size() == 1 && "NONE".equals(list.get(0))) {
-            return Collections.emptyList();
-        }
-        return list;
+        this.log.info("Configured audit settings: {}", auditConfig.toString());
```

Comment:
do not use `this`. Usually, `log` is a class variable, not an instance variable.

Reply:
Will make logger a class static

Reply:
Since log is used in subclasses. Will not make it static as it needs reference to class.

Reply:
Please don't use `this`, so that if later `log` is changed to a class variable, it won't be necessary to introduce change here.